doc/getting-started/quick_start.ipynb

```diff
@@ -455,21 +621,21 @@
    "id": "7b0edb0f-df47-4e6c-8c37-5f32c4bd6b39",
    "metadata": {},
    "source": [
-    "More detailed docs about Data conversion can be found [here](../data/data-conv.md)."
+    "More detailed docs about Data conversion can be found [here](https://docs.deepmodeling.org/projects/deepmd/en/master/data/data-conv.html)"
```

Comment:
We should use the relative links in the documentation without the hard-coded branch. The original one is correct.